demos/continuous_batching/README.md

```diff
@@ -195,22 +230,53 @@ for chunk in stream:
 
 Output:
 ```
-This is a test.
+It looks like you're testing me!
+```
+
+A similar code can be applied for the completion endpoint:
+```bash
+pip3 install openai
+```
+```python
```

Comment:
if used with echo command as in: https://github.com/openvinotoolkit/model_server/blob/main/demos/optical_character_recognition/python/README.md?plain=1#L45

expected output could be checked